plugins/API/SegmentMetadata.php

```diff
@@ -55,9 +62,21 @@ public function getSegmentsMetadata($idSites = array(), $_hideImplementationData
             $segments[] = $segment->toArray();
         }
 
+        $categoryList = CategoryList::get();
+
         foreach ($segments as &$segment) {
+            $categoryId = $segment['category'];
```

Comment:
Fixed segments were not sorted by categories as we do it in the menu, dashboard selector etc